docs webhooks (PR from TinaCMS) (#213)
SSWConsulting/SSW.Products: content/docs/EagleEye/webhooks.mdx

---
title: Webhooks
date: '2025-04-29T14:00:00.000Z'
---

> ## What are webhooks

Reviewer: ```suggestion
## What are webhooks?
```

> A webhook is a way for an application to send real-time data to another system when a specific event occurs. It works by making an HTTP request—usually a POST—to a designated URL with relevant information. Unlike APIs, which require constant polling to check for updates, webhooks push information automatically as events happen.

Reviewer: ```suggestion
A webhook is a way for an application to send real-time data to another system when a specific event occurs. It works by making an HTTP request — usually a `POST` — to a designated URL with relevant information. Unlike APIs, which require constant polling to check for updates, webhooks are event driven and push information automatically as events occur.
```

> In the context of EagleEye, our AI-powered email scanner, webhooks are triggered when EagleEye detects meaningful patterns in email content—such as task assignments, client updates, or urgent issues. For instance, when EagleEye scans an email and identifies a list of tasks assigned to you, it can immediately trigger a webhook to take action—like creating tasks, moving the email to a priority folder, or notifying your team. This allows you to integrate EagleEye seamlessly into your workflow and automate routine responses across platforms like Power Automate, Dynamics 365, or your project management system.

Reviewer: ```suggestion
In the context of EagleEye, webhooks are automatically triggered when predefined rules detect meaningful patterns in email content—such as task assignments, client updates, or urgent issues. Once a rule is configured in the Admin Portal, EagleEye scans incoming emails in user-defined intervals. When a match is found, the corresponding webhook is fired, enabling automated actions like creating tasks, prioritizing messages, or alerting your team. This seamless integration lets you automate routine workflows across platforms like Power Automate, Dynamics 365, or your preferred project management tools.
```

> ## Why using webhooks

Reviewer: ```suggestion
## Why use webhooks?
```
